README.md

```diff
@@ -1,13 +1,38 @@
 [![Build Status](https://travis-ci.org/ofiwg/libfabric.svg?branch=master)](https://travis-ci.org/ofiwg/libfabric)
 
-fabtests
-========
+# fabtests
 
-Fabtests provides a set of examples that uses [libfabric](https://github.com/ofiwg/libfabric) - a high-performance fabric software library.
+Fabtests provides a set of examples that uses
+[libfabric](https://github.com/ofiwg/libfabric) -a high-performance fabric
+software library.
 
-To make this directory, run:
+## Building fabtests
 
-    ./autogen.sh && ./configure && make && make install
+Fabtests uses GNU autotools. A typical install looks like:
+
+```
+./autogen.sh && ./configure && make && make install
+```
```

Comment:
Replace with:

> For example:
```sh
$ ./configure --with-libfabric=/opt/libfabric --prefix=/opt/fabtests && make -j 32 && sudo make install
```
This will tell the Fabtests to look for Libfabric libraries in the `/opt/libfabric` tree, and to install the Fabtests in the `/opt/fabtests` tree.  Alternatively:
```sh
$ ./configure --prefix=/opt/fabtests && make -j 32 && sudo make install
```
> Tells the Fabtests that it should be able to find the Libfabric header files and libraries in default compiler / linker search paths (configure will abort if it is not able to find them), and to install Fabtests in `/opt/fabtests`.